### DISCUSSION TITLE
Roll call - how many left?

### DISCUSSION THREAD

**Discussion by valnar1** [OpenSageTV/Community#11] [General]
On this new glorious chapter, it might be worth taking inventory on all the remaining active SageTV users.  Is there a way we can tell how many of us are left, other than everybody simply replying to this thread?

I saw the Sage locator service will be shut down soon.  I was wondering if that kept tabs on active users?

**Comment by wayner9** [comment]
I am still here and going strong.  I don't use locator at all.  If I use Sage remotely, which is rare, I was more likely to use a VPN or open up ports.

I have two unRAID servers running SageTV on a regular basis using Hauppauge HD-PVRs connected to HD cable boxes plus HD-HR OTA tuners.

**Reply by LoisGNS** [reply]
Still here - they'll have to pry my Sage from my achy remote-control-operating fingers.

**Comment by jusjoken** [comment]
The site tracks some things.  We have had 45 unique visitors to the discussions so far as an example.

Not sure if the stats are available to all but check out this link...

https://github.com/OpenSageTV/Community/graphs/traffic

**Reply by Narflex** [reply]
That link just sends me back to the github home page

**Reply by RJ-Make** [reply]
> That link just sends me back to the github home page

Same here

**Reply by techiegirl2day** [reply]
Wowzers!  On old forum went by Galaxysurfer.  I went to forum looking for help with epg since it is that time of year to renew the data feed & ended up here. I already had github account I was using for troubleshooting & giving feedback. (Not much of a developer.)  I missed a lot while was away! Currently running one server on Win 10  Sage 9.26 with a Hdhomerun Flex 4K. It is good to know there are still a few of us around keeping the home fires burning.

**Reply by techiegirl2day** [reply]
Seems to be just us Canucks so far.

**Comment by jusjoken** [comment]
Perhaps only available for owners.  Here is what it shows...

![image](https://github.com/OpenSageTV/Community/assets/6617820/314ba9b0-43f9-4632-a930-e83ab8e59cf4)

**Comment by jusjoken** [comment]
BTW: kinda cool you can just paste an image into the reply window and the image displays ;o)

**Reply by wayner9** [reply]
The advantages of modern discussion forum software!

**Comment by andyvshr** [comment]
Also here & still using SageTV daily. Too busy dealing with a termite infestation and pool repair lately, but I've at least been watching what's going on.

**Comment by JustFred51** [comment]
aka JustFred on the Sage forums.  Still here and loving Sage.  In the process of upgrading one of my Sage systems: removing the 2 (miserable) HdPvr capture boxes and replacing them with 2 Uray network encoders & OpenDCT.  Bought the network encoders on AliExpress for about $100 ea.  Same exact hardware that the Amazon Uray store sells for nearly twice that amount.

**Reply by wayner9** [reply]
Any issues with getting the Uray encoders working in SageTV?  I still have HD-PVRs but I am considering this path for the future.

**Reply by valnar1** [reply]
I didn't know these things existed myself.  I still have two older HD-PVR's and they work great.  Kinda afraid to touch them after all this time.

Conceptually I like the idea of capturing the digital signal, but the component capture doesn't bother me because these aren't shows I'm keeping for posterity.

**Reply by wayner9** [reply]
The bigger issue these days is getting devices with Component outputs.  Most newer equipment only has HDMI, and these Uray type of units take HDMI in and don't care about HDCP.  The main downside seems to be that they only capture stereo audio, but that isn't a big deal for me.

**Comment by sic0048** [comment]
Still around.....at least until Spectrum kills off cable cards.

**Comment by RJ-Make** [comment]
I'll be here until my SageTV installation stops working. I'll close the door behind me.. :-)

**Reply by valnar1** [reply]
I'll probably cancel Cable before I use another PVR.  That day might be coming though.

**Reply by Thomptronics** [reply]
Ditto. Been using SageTV since 2009 and no plans to stop until it does.

**Comment by jbuszkie** [comment]
I still use Sage all the time and will continue to do so until that cable cards stop working or Comcast cuts them off or we cancel cable all together.  I use Sage less and less.  All my kids just stream these days.  I mostly use Sage for live sporting events now.  The wife still watches some of the stuff that's being recorded..

**Comment by valnar1** [comment]
I didn't see the "Poll" section.  I guess I could have put this there...so I did! Head on over and cast a vote.  I will close this thread.

Link: https://github.com/OpenSageTV/Community/discussions/18

**Comment by Ugadata** [comment]
Still using SageTV daily.  CableCard- 1 HDHR Prime for Comcast.  1 HDHR Flex for OTA. Started experimenting with Channels DVR in order to eventually move off of Cable.  Not easy getting all the stuff you want from just one source.

Use AnyStream to get content from supported streamers.  Import into Sage to watch.  This is probably where most of the content I watch is at now, but not all.
